packages/core/src/index.ts

```diff
@@ -7,10 +7,10 @@ export type {
 export { executeRequest } from './http-client.js';
 export { getAiCoreDestination } from './context.js';
 export { OpenApiRequestBuilder } from './openapi-request-builder.js';
-export type {
+export {
   AzureOpenAiChatModel,
   AzureOpenAiEmbeddingModel,
   GcpVertexAiChatModel,
   AwsBedrockChatModel,
-  ChatModel
+  AicoreOpensourceChatModel
```

Comment:
```suggestion
  AiCoreOpensourceChatModel
```

[req] Please adjust in other places as well.

Reply:
I first had the same thought as yours as well until I saw they call it `aicore-opensource` and uses something like `implementation: AicoreOpensourceProvider` everywhere:

- https://github.tools.sap/MLF/mlf-gitops/blob/ee03d0311c4457ead7831a6ca99e4f13a8a4c118/docs/gen_ai_hub/README.md
- https://github.tools.sap/MLF/mlf-gitops/blob/ee03d0311c4457ead7831a6ca99e4f13a8a4c118/docs/gen_ai_hub/model-entry-catalogue.md#meta--llama3-70b-instruct

But true, we already used the term `OpenAi`...

Reply:
Done.